docs/first-use.rst

```diff
@@ -38,221 +37,60 @@ You should see something like this:
+If you want to get a feel for using CrateDB to work with time series data, you
+are going to need a source of time series data. Fortunately, there are many
+ways to generate time series data by sampling the systems running on your local
+computer.
 
-.. code-block:: text
+The :ref:`next collection of tutorials <gen-ts>` shows how to generate mock
+time series data about the International Space Station (ISS) and write it to
+CrateDB using the client of your choice.
 
-   HTTP/1.1 200 OK
-   content-length: 71
-   content-type: application/json; charset=UTF-8
 
-   {
-       "cols": [
-           "count(*)"
-       ],
-       "duration": 11.847271,
-       "rowcount": 1,
-       "rows": [
-           [
-               3879
-           ]
-       ]
-   }
+.. _start-building:
 
-In this example, we have a single row with a single column: ``3879``.
+Start building with CrateDB clients and tools
+=============================================
 
+If you'd like to skip the tutorials and start building with CrateDB, you can
+find `a list of available CrateDB clients and tools`_ in a different section of
+the documentation.
 
-Next steps
-==========
 
-Start building with `CrateDB clients and tools`_.
+.. _dive-in:
 
-.. SEEALSO::
+Dive in to CrateDB
+==================
 
-    :ref:`Generate time series data <gen-ts>` (tutorials for multiple
-    languages)
+Check out the `How-to Guides`_ for goal oriented topics. Alternatively, check
+out the `CrateDB Reference`_ for a complete dip-in-and-dip-out reference manual.
 
 
-.. _admin UI: https://crate.io/docs/clients/admin-ui/en/latest/
+.. _a list of available CrateDB clients and tools: https://crate.io/docs/crate/clients-tools/en/latest/
+.. _Admin UI: https://crate.io/docs/clients/admin-ui/en/latest/
 .. _aggregation: https://crate.io/docs/stable/sql/aggregation.html
 .. _client tools: https://crate.io/a/category/client-tools/
 .. _Crash: https://crate.io/docs/clients/crash/en/latest/
-.. _CrateDB clients and tools: https://crate.io/docs/crate/clients-tools/en/latest/
+.. _CrateDB Reference: https://crate.io/docs/crate/reference/
 .. _fulltext search: https://crate.io/docs/crate/reference/en/latest/general/dql/fulltext.html
 .. _geospatial: https://crate.io/docs/stable/sql/data_types.html#geo-point
+.. _How-to Guides: https://crate.io/docs/crate/howtos/
 .. _HTTP endpoint: https://crate.io/docs/crate/reference/en/latest/interfaces/http.html
 .. _HTTPie: https://httpie.org/
 .. _install Crash: https://crate.io/docs/clients/crash/en/latest/getting-started.html#install
 .. _object: https://crate.io/docs/crate/reference/en/latest/general/ddl/data-types.html#object
 .. _regular SQL: https://crate.io/docs/crate/reference/en/latest/appendices/sql-compliance.html
 .. _scalar: https://crate.io/docs/stable/sql/scalar.html
 .. _schema: https://crate.io/docs/crate/reference/en/latest/general/ddl/create-table.html#schemas
+.. _The CrateDB Admin UI documentation: https://crate.io/docs/clients/admin-ui/en/latest/
```

Comment:
```suggestion
.. _The CrateDB Admin UI documentation: `Admin UI`_
```

Reply:
woah I had no idea this was possible. I love it! very meta. haha